includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -2257,45 +2291,6 @@ private function needs_ssl_setup() {
 		return ! $this->testmode && ! is_ssl(); // @phpstan-ignore-line (testmode is defined in the classes that use this class)
 	}
 
-	/**
-	 * Helper method to retrieve the status of the order before it was put on hold.
-	 *
-	 * @since 8.3.0
-	 *
-	 * @param WC_Order $order The order.
-	 *
-	 * @return string The status of the order before it was put on hold.
-	 */
-	protected function get_stripe_order_status_before_hold( $order ) {
```

Comment:
This function is still being used in includes/class-wc-stripe-webhook-handler.php.

Reply:
Good catch! I have reverted the removal in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4106/commits/9e9b0092b130ae702507a464b1e96c8a7bdb76e0